src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -267,8 +263,8 @@ public static void TestDecryptSimpleTripleDes_IssuerAndSerial()
         }
 
         [Fact]
-        [ActiveIssue(3334, PlatformID.AnyUnix)]
         [OuterLoop(/* Leaks key on disk if interrupted */)]
+        [ActiveIssue(3334, PlatformID.AnyUnix)]
```

Comment:
Presumably you removed the attribute because it should have worked, then put it back because it didn't... why doesn't this one work?